indigo/distribution.yaml

```diff
@@ -12877,6 +12877,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_moveit.git
       version: master
     status: maintained
+  rqt_mrta:
+    doc:
+      type: git
+      url: https://github.com/adrianohrl/rqt_mrta.git
+      version: 0.2.5
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/rqt_mrta-release.git
+      version: 0.2.5-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/adrianohrl/rqt_mrta.git
+      version: 0.2.5
```

Comment:
This reference needs to be a branch, Jenkins doesn't support tracking tags for the CI.